postcss.config.js

```diff
@@ -16,7 +21,17 @@ module.exports = {
             },
         },
         'postcss-flexbugs-fixes': {},
-        // 'postcss-rtl': {},
-        // cssnano: {},
+        'postcss-rtl': {},
+        'postcss-modules': {
+            getJSON: function(cssFileName, json, outputFileName) {
+                const cssName = path.basename(cssFileName, '.css')
+                const cssDir = path.dirname(cssFileName).split('src/')[1]
+                const jsonFileName = path.resolve(
+                    './build/' + cssDir + '/' + cssName + '.json'
+                )
+                fs.writeFileSync(jsonFileName, JSON.stringify(json))
```

Comment:
Here we can do static parsing of the JSX files and replace the classes which are not computed runtime.